src/python/bot/fuzzers/libFuzzer/engine.py

```diff
@@ -101,12 +101,13 @@ def prepare(self, corpus_dir, target_path, _):
       A FuzzOptions object.
     """
     arguments = fuzzer.get_arguments(target_path)
+    grammar = environment.get_value('LIBFUZZER_GRAMMAR', None)
```

Comment:
Why was this changed to an env var? 

There are a number of issues with this:
1. This doesn't actually propagated anywhere for libFuzzer. There is a [whitelist](https://github.com/google/clusterfuzz/blob/5e9b02204428cb1a81993d512844966d2bee0a80/src/python/bot/fuzzers/options.py#L36), and the propagation currently only happens for [AFL](https://github.com/google/clusterfuzz/search?q=get_env&unscoped_q=get_env).

2. Using env vars for this seems a bit hacky -- env vars are very general, pollute the namespace of multiple processes, and we should avoid adding any more to our (already very large list) if we can.. An explicit section in the options file (or the .grammar) is better.

Reply:
We thought that it would get piped in from the environment variables section of the libfuzzer options file without any need for a change on the chromium side. These are good points though, so now it works with a separate grammar section in the libfuzzer options file.